examples/js/loaders/GLTFLoader.js

```diff
@@ -2621,11 +2621,20 @@ THREE.GLTFLoader = ( function () {
 				cachedMaterial = material.clone();
 
 				if ( useSkinning ) cachedMaterial.skinning = true;
-				if ( useVertexTangents ) cachedMaterial.vertexTangents = true;
 				if ( useVertexColors ) cachedMaterial.vertexColors = true;
 				if ( useFlatShading ) cachedMaterial.flatShading = true;
 				if ( useMorphTargets ) cachedMaterial.morphTargets = true;
 				if ( useMorphNormals ) cachedMaterial.morphNormals = true;
+				
+				if ( useVertexTangents ) {
+
+					cachedMaterial.vertexTangents = true;
+
+					// https://github.com/mrdoob/three.js/issues/11438#issuecomment-507003995
+					if ( material.normalScale ) material.normalScale.y *= -1;
+					if ( material.clearcoatNormalScale ) material.clearcoatNormalScale.y *= -1;
```

Comment:
These two lines are needed only if vertex tangents are _not_ present — that's the issue in https://github.com/mrdoob/three.js/issues/21136 as well.

Reply:
No, because I reversed them by default above, so now they need to be re-reversed if vertexTangents are used, which can work because vertexTangents create a new cacheKey.

Reply:
Oops, I see. Seems a little more confusing to have inversions in three places, was there a specific reason you prefer that? If we keep it that way, we should include a link to the github issue above in all three places.

Reply:
As I say, it was necessary in order to make the switch happen only inside a cached constructor, instead of every time `assignFinalMaterial` was called. That was the bug. I'll put in the comments.

Reply:
It seems like this will introduce the opposite problem if the mesh did have vertex tangents with variants, no?

Reply:
No, I don't think so. Did you find that to be true in testing?

Reply:
We have exactly one sample using variants, and it doesn't have vertex tangents, so I'm limited to guessing at this point. I think I see what you mean though.